app/src/main/res/values/strings.xml

```diff
@@ -2,6 +2,9 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <resources>
+    <!-- App name for private browsing mode, only the "(Private)" portion should be localized. -->
+    <string name="app_name_private">Firefox Preview (Private)</string>
```

Comment:
We should use placeholder text for the app_name like we do here: https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/values/strings.xml#L24

Reply:
This string is only used in the app manifest file, and I'm not sure how we can make placeholder text work with `AndroidManifest.xml`.

Reply:
👍